Dockerfile

```diff
@@ -13,11 +13,16 @@ RUN yum -y --repo ubi-9-appstream-rpms install socat && \
   unzip -qoj acme.zip acme.sh-${ACME_RELEASE_TAG}/acme.sh -d . && rm acme.zip && \
   echo "ACME=${ACME_RELEASE_TAG}" >> versions && echo "DRIVER=${DRIVER_RELEASE_TAG}" >> versions
 
+RUN rpm --import https://packages.microsoft.com/keys/microsoft.asc && \
```

Comment:
I think I recall Kevin saying that Azure CLI is installed just in case we need it during development, we should put a comment here so we don't forget to remove Azure CLI once this is working.

Reply:
If we ad a TODO, this can be controlled via a build arg as well for the Dockerfile. Adding the Azure CLI to the image, supports running the image locally with the developers CLI security context, after `az login` on their local machine.

> Leaving the CLI installed on the image, however, doesn't affect anything in production, as it's inert. If the image runs as a container with a user or system managed identity, that identity will be granted the ability to take action on whatever Azure resources its been granted permission on.

Reply:
Let's add a comment here that it is only for local development and we can come back to this at a later point if we need to.